functions/functions/messages/list-messages.js

```diff
@@ -0,0 +1,12 @@
+const { listMessages } = require(Runtime.getFunctions()["twilio/messages"].path);
+
+exports.handler = (context, event, callback) => {
+    const client = context.getTwilioClient();
+    const { filterCriteria } = event;
+    listMessages(client, filterCriteria)
+    .then(data => callback(null, data))
+    .catch(err => {
+        console.log(`Error getting account messages ${err}`);
```

Comment:
I think the console.log here is duplicating what will be sent to the debugger. And the code isn't actively being worked on, so shouldnt include in the commit